Object_list:
  Object:
    Cleaning_supplies:
      sponge: 50
      clanser: 500

    Toys:
      tennis_ball: 58 
      soccer_ball: 430
      dice: 8
      baseball: 145
      rubiks_cube: 98

    Dishes:
      fork: 30
      knife: 80
      spoon: 50
      plate: 500
      bowl: 300
      cup: 250
      bottle: 300

    Drinks:
      red_wine: 1200
      cola: 380
      juice_pack: 200
      tropical_juice: 1200
      milk: 1000
      iced_tea: 340
      orange_juice: 1200

    Food:
      cereal: 500
      tuna: 165
      spam: 340
      mustard: 250
      strawberry_jello: 85
      chocolate_jello: 85
      coffee_grounds: 250
      sugar: 1000

    Fruits:
      apple: 150
      orange: 200
      pear: 180
      banana: 120
      strawberry: 10
      lemon: 110
      peach: 150
      plum: 80

    Snacks:
      pringles: 150
      cornflakes: 500
      cheezit: 200